### ISSUE TITLE
ACS 4.19 | No Auth Failure Message Displayed when login Password is wrong

### ISSUE THREAD

**Issue by midhunpjos** [apache/cloudstack#8662]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
API, UI
~~~

##### CLOUDSTACK VERSION

~~~
4.19
~~~

##### CONFIGURATION

CS 4.19 Advanced Zone, Normal User Password Authentication

##### OS / ENVIRONMENT

Ubuntu 20,04

##### SUMMARY

When Logging in via UI if the password is wrong auth failure response is not received. The UI Progress Bar Keeps on Loading
##### STEPS TO REPRODUCE

~~~
Log In to UI, Give Correct Username and Incorrect Password. 
Auth Failure Response is not received.
~~~

##### EXPECTED RESULTS

~~~
If Password is Wrong we should get a message as " Failed to authenticate user"
~~~

##### ACTUAL RESULTS

~~~
No Auth Failure response is received
~~~

**Comment by DaanHoogland** [comment]
@midhunpjos can you describe what you receive, or happens otherwise. I hope you can not login with the wrong password.

**Comment by vishesh92** [comment]
I am able to reproduce this.
While logging in, if the user doesn't exist it fails with a valid error
![image](https://github.com/apache/cloudstack/assets/8760112/35ffd5e2-f5e2-44d3-91f5-c209b44f9a89)

If the user exists, but the password is wrong then loading starts on top and the UI gets stuck. I had to clean up my local storage to make the browser work again.
![image](https://github.com/apache/cloudstack/assets/8760112/b542af8f-7e2e-4d86-ab88-3ae9a400adac)

**Comment by rohityadavcloud** [comment]
Fixed in https://github.com/apache/cloudstack/pull/8671

**Comment by rohityadavcloud** [comment]
cc @vishesh92 pl check and re-open if I made an error in closing this?

**Comment by rohityadavcloud** [comment]
My bad, Vishesh advised me PR needs more work to merge. Re opening the issue.

**Comment by DaanHoogland** [comment]
Fixed by #8671
